main.py

```diff
@@ -2650,5 +2650,30 @@ async def recieve_vote(request):
     await claim_reward(user, channeley, type)
     return web.Response(text="ok", status=200)
 
+@bot.tree.command(name="cat_fact", description="get a random cat fact")
+async def cat_fact(message: discord.Interaction):
+    
+    facts = [
+        "you love cats",
+        "cat bot is in 23,805+ servers",
```

Comment:
i think it would be cool to put the actual amount here `len(bot.guilds)`

Reply:
thanks i didnt know how to do it